docs/geedocs/5.3.2/answer/176738.html

```diff
@@ -309,6 +309,9 @@ <h2 id="status_request_timeout">Status request timeout</h2>
    spend waiting to fulfill a status request before it times out. This timeout can be configured manually by setting
    <code>MutexTimedWaitSec</code> in <code>/gevol/assets/.config/misc.xml</code></p>
 <code>&lt;MutexTimedWaitSec&gt;60&lt;/MutexTimedWaitSec&gt;</code>
+<br><br>
+<h2 id="utilizing_graph_operations_in_resume_operations">Utilizing graph operations in resume operations</h2>
+<p>Included in <code>/gevol/assets/.config/misc.xml</code> is a <code>GraphOperations</code> option, and it may be set to <code>true</code> or <code>false</code>. It is an experimental operation which allows for resume operations to be optimized via graph operations.  
```

Comment:
My thoughts:
- I think it needs to be set to `1` or `0`, not `true` or `false`.
- I'm not sure that we need to mention that it uses graph operations, as the average user won't care about that. We can just say it's an experimental version that improves performance.
- I'd like to make the experimental part clearer by putting it into its own sentence.
- The GraphOperations option will likely be used for more than just resume as we update more operations.

So the second sentence would get split into two sentences and look something like this: "It enables optimized versions of some operations. These optimizations are experimental and may contain bugs."